docker-compose.yml

```diff
@@ -2,8 +2,6 @@ db:
   image: mongo:latest
 web:
   build: .
-  working_dir: /vagrant/oratory/
-  command: supervisor server.js
```

Comment:
@mynameisfiber it seemed like it was redundant having these here. Correct me if I'm wrong.